### ISSUE TITLE
Area graph support in dashboards

### ISSUE THREAD

**Issue by makeavish** [SigNoz/signoz#2889]
New relic supports area graph and one of user has requested support for area chart.

Example:
![image](https://github.com/SigNoz/signoz/assets/9512100/f9ea411d-c349-44af-9eca-84b3a432a14c)

**Comment by palashgdev** [comment]
Sure @Damans227

**Comment by Damans227** [comment]
Hi @makeavish I am working on the menu item for add panel in the dashboard. Where can I find the base64 area chart icon? As of right now, [this is](https://www.iconfinder.com/icons/1608513/area_chart_icon) what I'm using.

**Comment by makeavish** [comment]
> Hi @makeavish I am working on the menu item for add panel in the dashboard. Where can I find the base64 area chart icon? As of right now, [this is](https://www.iconfinder.com/icons/1608513/area_chart_icon) what I'm using.

@Damans227 You can use [AntD icons](https://ant.design/components/icon).  Eg: `<AreaChartOutlined />`

**Comment by makeavish** [comment]
@Damans227 Are you still working on this?

**Comment by Damans227** [comment]
Hi @makeavish Yes, I am. I am facing an issue bringing up click house docker service and asked for help in the slack channel. 

https://signoz-community.slack.com/archives/C027134DM8B/p1688397690606119?thread_ts=1688397690.606119&cid=C027134DM8B

Just waiting for a response. Can you take a look? I think it will be easier if someone can send me a test url which I can use.

**Comment by makeavish** [comment]
> 

Sent you test url in community.

**Comment by makeavish** [comment]
> Hi @makeavish Yes, I am. I am facing an issue bringing up click house docker service and asked for help in the slack channel.
> 
> https://signoz-community.slack.com/archives/C027134DM8B/p1688397690606119?thread_ts=1688397690.606119&cid=C027134DM8B
> 
> Just waiting for a response. Can you take a look? I think it will be easier if someone can send me a test url which I can use.

Sent you test URL in community.

**Comment by Damans227** [comment]
Hey @makeavish, just a heads up, I am now looking into `chart.js` to implement the area chart. In order to do so, I am using the existing [ line graph ](https://github.com/Damans227/signoz/tree/develop/frontend/src/components/Graph) as an example.

Also, it looks like we may need to add an area panel type to the query range API in order to make this work. If you have any suggestions, please let me know. Currently, I am testing using a line graph panel type.

**Comment by palashgdev** [comment]
possibly i am thinking query would be there

@makeavish can you please share the query

**Comment by Damans227** [comment]
Ptal [here](https://signoz-community.slack.com/archives/C027134DM8B/p1691426146317659).

**Comment by makeavish** [comment]
> possibly i am thinking query would be there
> 
> @makeavish can you please share the query

@Damans227 Area chart query and response will be the same as the time series. It's just that line charts are stacked in the UI.
Push your changes with panelType graph for now, we will review.

**Comment by ahrefabhi** [comment]
@makeavish We have added a support for fill mode in Timeseries panel in dashboard v2, can you tell me what's the difference between area chart / timeseries(filled)?

or will the below changes fulfil our needs?

https://github.com/user-attachments/assets/5faba3da-3576-469d-9f7a-b19a9b6bb7e2

**Comment by makeavish** [comment]
> [@makeavish](https://github.com/makeavish) We have added a support for fill mode in Timeseries panel in dashboard v2, can you tell me what's the difference between area chart / timeseries(filled)?
> 
> or will the below changes fulfil our needs?
> 
>  Screen.Recording.2026-07-29.at.3.07.03.PM.mov

Yes, this should fulfil the original requirement. An area chart uses the same query and response as a time-series chart; the difference is primarily the filled visualization. We don’t need a separate area panel type.  
Let’s just verify that multiple series can be stacked, since that may be part of the expected area-chart behavior. If stacking works as expected, we can consider this issue resolved. Or maybe create a new issue for stacking.

**Comment by ahrefabhi** [comment]
What do you mean by stacking @makeavish ?
